buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateEapBuildNumber.kt

```diff
@@ -0,0 +1,69 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+import org.gradle.util.VersionNumber
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.select.Elements
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+open class UpdateEapBuildNumber : DefaultTask() {
+
+    @Internal
+    var intellijSnapshotsUrl: String = "https://www.jetbrains.com/intellij-repository/snapshots/"
+
+    @Internal
+    @Option(option = "exit-code", description = "Return the exit code 0 when the new eap build number is found and 1 otherwise")
```

Comment:
While you're here
```suggestion
    @Option(option = "exit-code", description = "Return the exit code 0 when a new EAP build number is found and 1 otherwise")
```

Reply:
Or even better:
```suggestion
    @Option(option = "exit-code", description = "If set to true, the task will return the exit code 1 when the new EAP build number is NOT found")
```

Coz that's the actual difference in behavior